guac-visualizer.md

```diff
@@ -27,83 +27,56 @@ well as a means to explore the supply chain and prototype policies.
 ## Requirements
 
 - [Git](https://git-scm.com/downloads)
-- [Yarn](https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable)
-- [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+- [yarn](https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable)
+- [node](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
 - A fresh copy of the [GUAC service infrastructure through Docker Compose]({{
   site.baseurl }}{%link setup.md %}).
 
-## Step 1: Getting started
+# 1. Getting started
 
-1. If you haven't already, download the main GUAC version `v0.1.2` from
-   [this link](https://github.com/guacsec/guac/releases/tag/v0.1.2).
-   - Follow steps 2-5 in [setup]({{ site.baseurl }}{% link setup.md %}).
-2. Download the specific GUAC visualizer version `v0.1.1` from
-   [this link](https://github.com/guacsec/guac-visualizer/releases/tag/v0.1.1)
-   and extract it in the same path as your GUAC repo.
+To use the GUAC visualizer, you need to have the main GUAC server running.
 
-Change directories into the visualizer's repo:
-
-```bash
-cd path/to/guac-visualizer
-```
+### 1. Download GUAC version `v0.2.0` from [here](https://github.com/guacsec/guac/releases/tag/v0.2.0).
 
-All commands run throughout this guide should be in this working directory.
+- Follow these steps outlined here to get the GUAC server up and running:
+  https://docs.guac.sh/setup
```

Comment:
Should be linked like this `[link text here]({{ site.baseurl }}{% link setup.md %})`